docs/source/installing-brave.rst

```diff
@@ -14,6 +14,21 @@ Release Channel Installation
 
 .. highlight:: console
 
+Debian
+--------------------------
+::
+
+    curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -
+
+    source /etc/os-release && CODENAME=$(echo $VERSION | rev | cut -c 2- | rev | cut -d"(" -f 2)
```

Comment:
Missing paren here

Reply:
I don't see it?

Reply:
I guess, you could just use `$VERSION_CODENAME` here, that in my case is just `buster`